quupe-RN/js/screens/Messages/Messages.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+const Messages = () => {
+    return (
+        <View>
+            <Text> Favorites </Text>
```

Comment:
this is a message component. Make sure you have right text showing up on the screen